### ISSUE TITLE
Exception message grammar: "An SqlParameter [...] is not contained by this SqlParameterCollection"

### ISSUE THREAD

**Issue by jnm2** [dotnet/SqlClient#159]
Repro:

```cs
using (var command = new SqlCommand(null, null))
{
    _ = command.Parameters["@foo"];
}
```

> System.IndexOutOfRangeException: An SqlParameter with ParameterName '@​foo' is not contained by this SqlParameterCollection.

Expected: "**A** SqlParameter," not "**An** SqlParameter"

Source:

https://github.com/dotnet/corefx/blob/6739eab2f4ee9a895575289893edf63b6c234d43/src/System.Data.SqlClient/src/Resources/Strings.resx#L122-L124

**Comment by v-kaywon** [comment]
@jnm2 Attached is the driver built from PR #252 containing the fix. Can you verify if the fix is ok?

[Microsoft.Data.SqlClient.1.1.0-dev.zip](https://github.com/dotnet/SqlClient/files/3713584/Microsoft.Data.SqlClient.1.1.0-dev.zip)

**Comment by v-kaywon** [comment]
Closing this as the fix in PR #252 is merged.
